sources/macro.h

```diff
@@ -22,4 +22,8 @@
 
 typedef int od_retcode_t;
 
+/*misc*/
+
+#define INVALID_ID -1
```

Comment:
Maybe INVALID_PID?

Reply:
this is essentially either thread id orr mm coroutine id so ... just id